crates/aptos/e2e/common.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Aptos
+# SPDX-License-Identifier: Apache-2.0
+
+from dataclasses import dataclass
+from enum import Enum
+
+
+NODE_PORT = 8080
+FAUCET_PORT = 8081
+
+
+class Network(Enum):
+    DEVNET = "devnet"
+    TESTNET = "testnet"
+    MAINNET = "mainnet"
+
+    def __str__(self):
+        return self.value
+
+
+# Information for some accounts we use for testing.
+@dataclass
+class AccountInfo:
+    private_key: str
+    public_key: str
+    account_address: str
+
+
+ACCOUNT_ONE = AccountInfo(
+    private_key="0x37368b46ce665362562c6d1d4ec01a08c8644c488690df5a17e13ba163e20221",
```

Comment:
We should in the future use credentials in a credential manager rather than hardcoding keys

Reply:
How come? This key is only used in these tests, which only work against local testnets.